wai-aria/scripts/aria-utils.js

```diff
@@ -124,13 +124,13 @@ const AriaUtils = {
         data-expectedlabel="foo"
         class="ex">
 
-      AriaUtils.verifyLabelsBySelector(".ex")
+      AriaUtils.verifyLabelsBySelector(1, ".ex")
 
   */
-  verifyLabelsBySelector: function(selector) {
+  verifyLabelsBySelector: function(expectedCount, selector) {
     const els = document.querySelectorAll(selector);
-    if (!els.length) {
-      throw `Selector passed in verifyLabelsBySelector("${selector}") should match at least one element.`;
+    if (els.length !== expectedCount) {
+      throw `verifyLabelsBySelector expected ${expectedCount} elements but received ${els.length}`;
```

Comment:
This is going to break a few outstanding PRs that expect `selector` to be first... I also would prefer `selector` remain the first argument and throw a warning if a count is not provided. And the error only if it is provided but doesn't match.

`comp_label.html`, `comp_labelledby.html`, and `comp_text_node.html` could then be left out of this PR to avoid conflicts/breakage. They would throw a warning but not an error, and we could then update them in those PRs (or later ones) to use the optional-but-recommended `expectedCount`.